design/schema/bcdc.puml

```diff
@@ -0,0 +1,319 @@
+@startuml
+
+hide empty members
+'hide circle
+
+
+'Enums
+enum WebResourceType {
+    Download
+    Information
+    Explore
+    View
+}
+
+enum AccessControl {
+    Open
+    Embargoed
+    Controlled
+}
+
+enum CompletionState {
+    Complete
```

Comment:
There are additional states for triple store ingest and validation, do we want yto incorporate them here as well?

Reply:
Triple store ingest/validation states apply to individual uploads, and we can have multiple uploads per data collection/multiple data collections per upload. We do have an association table for uploads/data collections, but that is outside of this schema.